README_MACOS.md

```diff
@@ -179,79 +208,71 @@ Simply posting "the latest checkout is broken" won't help. We need the exact com
 Frequently used cmake settings
 ------------------------------
 
-There are more settings in the build configuration you may want to adjust. In order to
-see a useful list of your options, you can run:
+There are more settings in the build configuration you may want to adjust. In order to see a useful list of build options, you can run:
```

Comment:
Because (Github-?) md seems to keep line breaks in the source file. Looks like that in your quotation, doesn't it? Other than that the Readme was a big mix of paragraphs with line-breaks and others without. I think it's quite consistent now.